src/accounts/accounts-db/k8s/overlays/development/kustomization.yaml

```diff
@@ -14,8 +14,11 @@
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+commonLabels:
+  tier: db
 resources:
   - ../../base
   - accounts-db.yaml
 components:
-  - ../../../../../components/development
\ No newline at end of file
+  - ../../../../components/accounts
```

Comment:
Same comment as above

Reply:
hey @TheRealSpaceShip , please take a look

Reply:
@bourgeoisor components properties defined in `base` are not applied to Deployment manifests that is resource in `development` overlay of  `ledger-db` and `account-db`. That's why I duplicates [account-db component](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/src/accounts/components/accounts/kustomization.yaml) and [ledger-db component](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/src/ledger/components/ledger/kustomization.yaml) here as well as `commonLabels.tier: db`.
To compare the difference just run `skaffold render --profile=development --digest-source=none --module ledger-db` over `main` branch and this one. My version of skaffold is 2.7.1
If this is a bad fix for an issue with wrong labels I could revert `kustomization` files and leave only files related to backup to unblock this PR.